ISSUE_TEMPLATE/stable.md

```diff
@@ -86,5 +86,4 @@ NOTE: In the future, most of these steps will be automated.
 
 ## Open an issue for the next release
```

Comment:
Not new though should we also rename this "Housekeeping" to be consistent with the other templates?

Reply:
Done!